### PR TITLE
Update to use Protocol v18.0.0

### PR DESCRIPTION
**Pull request by nathan-barrett**
## One-line summary
Bumps version of protocol

## Testing
Run the following:
- make build
- make build-prod
You should not get any errors in the console

### CODE REVIEW

File: media/css/firefox/browsers/best-browser.scss
Comment (reemhamz) [inline media/css/firefox/browsers/best-browser.scss:1]: I'm still noticing font-size differences in this page?

**Review by reemhamz — CHANGES_REQUESTED** [review]
Nearly there!
I did notice that there were some `get-theme(..)` property values that haven't been updated in the following files:

- `welcome15.scss`
- `relay/landing.scss`
- `vpn/pricing-refresh.scss`

I would also like to point out that I'd like to work together to figure out why some Firefox brand themes are not applying as they should to some pages that I noted before:

> body-text-lg has changed from size 18px to size 17.6px (http://localhost:8000/en-US/firefox/browsers/best-browser/)
>     text-title-md went from size 48px to size 40px, which is a noticeable difference (http://localhost:8000/en-US/firefox/browsers/best-browser/)
>     text-title-sm went from size 38px to size 32px (look at callout at bottom: https://www.mozilla.org/en-US/firefox/browsers/windows-64-bit/)
>     text-title-xs went from size 28px to size 24px (http://localhost:8000/de/firefox/nightly/firstrun/)

**Comment by reemhamz** [comment]
Integration tests passed: https://github.com/mozilla/bedrock/actions/runs/5863768037/workflow

File: media/css/firefox/browsers/best-browser.scss
Comment (nathan-barrett) [reply media/css/firefox/browsers/best-browser.scss:1]: The value for `body-text-lg` was slightly incorrect, but for the other three values, it was actually an issue in production - they were using the mozilla brand values for font-sizes here.

In protocol 17.1, you can see [here](https://github.com/mozilla/protocol/blob/f318aafa0f3b5ff8815c4b859d5a2de9146657f4/assets/sass/protocol/includes/_themes.scss#L107) that the size for `text-title-md` should be 40px, not 48px

That `text-title-sm` [should be 32px](https://github.com/mozilla/protocol/blob/f318aafa0f3b5ff8815c4b859d5a2de9146657f4/assets/sass/protocol/includes/_themes.scss#L109)

that `text-title-xs` should [24px and not 28px](https://github.com/mozilla/protocol/blob/f318aafa0f3b5ff8815c4b859d5a2de9146657f4/assets/sass/protocol/includes/_themes.scss#L111)

So the new values are the _actual_ correct ones, and the values on production currently _incorrect_.

Comment (nathan-barrett) [reply media/css/firefox/browsers/best-browser.scss:1]: follow up: Just merged a fix for the `body-text-lg` value in protocol: https://github.com/mozilla/protocol/pull/894

**Comment by reemhamz** [comment]
Also tested things on demo and things seem good: https://www-demo3.allizom.org/en-US/

**Review by reemhamz — APPROVED** [review]
Spoke about the font-size changes and updates on a call, and things should be correct from now on since the current prod had incorrect font-sizes due to an import issue.

Will make a new issue addressing the Inter -> Metropolis titles in Firefox pages.

r+ 🎉‼️hopefully things don't crash this time.

**Pull request merged**